### DISCUSSION TITLE
Styling Collection with Siblings

### DISCUSSION THREAD

**Discussion by david-f** [jrief/django-formset#67] [Q&A]
I have been able to create a form for a Collection containing a Sibling. It has a Delete button and an Add button annotated with the add_label. However, the Delete and Add buttons appear to have no styling associated with them and I want to give them the same style as the main Submit button, where would I put this styling information?
Thanks in advance,
David

**Comment by YLITC** [comment]
The documentation mentions this at [14.3. Rendering Collections](https://django-formset.fly.dev/form-renderer/#rendering-collections). You can add a default_renderer to a collection.
